### DISCUSSION TITLE
Print file path report as a link file

### DISCUSSION THREAD

**Discussion by adef145** [detekt/detekt#5912] [Ideas]
Hi detekt team, is there any reason detekt didn't print report path as a link file? So we can click the path, and will open browser / text editor / etc?
 
![Screenshot from 2023-03-17 14-57-05](https://user-images.githubusercontent.com/9027262/225845992-c0749b96-dc66-4cfd-a9df-2629c874d18d.png)

**Comment by BraisGabin** [comment]
Could you elaborate? Where should detekt do that?

**Comment by cortinico** [comment]
I believe we already do what you're asking in the console report, but please provide more context as it's hard to understand

**Comment by adef145** [comment]
Like in my screenshoot, on `OutputFacade.handleOutputReports` we can add prefix `file://` before filePath.
![Screenshot from 2023-03-17 14-57-05](https://user-images.githubusercontent.com/9027262/225946766-09837005-29cd-4d08-b9fb-ca7f86536f0e.png)

So before we adding prefix `file://`
![WhatsApp Image 2023-03-17 at 9 36 39 PM](https://user-images.githubusercontent.com/9027262/225936998-501fe897-ab5c-4b01-a501-41f3f0e9de24.jpeg)
We only see path file for the reports, but we can not click it, because it's not a link.

But if we add prefix `file://`, we can see in our terminal like here
![WhatsApp Image 2023-03-17 at 9 38 25 PM](https://user-images.githubusercontent.com/9027262/225937401-7c437157-9afd-4101-bb80-38f65d67fc7c.jpeg)
* If we click it on html report, will redirect to browser
* if we click sarif report, will open sarif file inside Android Studio
* if we click markdown report, will open markdown file inside Android Studio

I already made a changes on my forking repository in [here](https://github.com/detekt/detekt/compare/main...adef145:detekt:main). If you guys don't mind, I will raise a PR

**Reply by cortinico** [reply]
Yes that's doable 👍 Please send over the PR

**Reply by adef145** [reply]
Alright, thanks

**Reply by adef145** [reply]
Hi @cortinico , please kind review my PR in [here](https://github.com/detekt/detekt/pull/5921).

Thanks for advance
